pkg/api/query/grpc.go

```diff
@@ -60,6 +62,11 @@ func (g *GRPCAPI) Query(request *querypb.QueryRequest, server querypb.Query_Quer
 		maxResolution = g.defaultMaxResolutionSeconds.Milliseconds() / 1000
 	}
 
+	lookbackDelta := g.lookbackDeltaCreate(maxResolution)
```

Comment:
Shouldn't it be 
```go
lookbackDelta := g.lookbackDeltaCreate(maxResolution * 1000)
```
since it's `lookbackDeltaCreate` (and old `queryEngine(..)`) suppose to receive parameter in `millis`?

Reply:
I think I made a mistake to use seconds in the gRPC API when the HTTP one uses millis. Because of this, I also think we should multiply by 1000.

Reply:
@fpetkovski what do you think about this please?

Reply:
done @fpetkovski 🙏